arch-wiki-docs.py

```diff
@@ -33,5 +34,12 @@
 
     downloader.download_images()
 
+    for lang_dir in next(os.walk(args.output_directory))[1]:
```

Comment:
Please use [os.listdir](https://docs.python.org/3/library/os.html#os.listdir) instead of `os.walk`.